.gitignore

```diff
@@ -9,6 +9,9 @@ lerna-debug.log*
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
+# Intellij IDEA configuration's folder
+.idea
```

Comment:
There's no reason to include this in this file as a global .gitignore should be the one excluding IDE-specific things :)

Reply:
I can remove it sure.
Added here, as it's quite common to exclude `.idea` in `.gitignore` :)